.ci/Jenkinsfile-compile

```diff
@@ -70,46 +70,46 @@ pipeline {
 
           // docker builds:
           def arch_builds = [
-            target: ["posix_sitl_default"],
+            target: ["px4sitl_default"],
             image: docker_images.arch,
             archive: false
           ]
 
           def armhf_builds = [
-            target: ["posix_ocpoc_ubuntu"],
+            target: ["aerotennaocpoc_ubuntu"],
             image: docker_images.armhf,
             archive: false
           ]
 
           def base_builds = [
-            target: ["posix_sitl_rtps"],
+            target: ["px4sitl_rtps"],
             image: docker_images.base,
             archive: false
           ]
 
           def nuttx_builds_archive = [
             target: ["px4fmu-v2_default", "px4fmu-v3_default", "px4fmu-v4_default", "px4fmu-v4pro_default", "px4fmu-v5_default", "px4fmu-v5_rtps", "px4fmu-v5_stackcheck",
-                     "aerofc-v1_default", "aerocore2_default", "auav-x21_default", "av-x-v1_default", "crazyflie_default", "mindpx-v2_default",
-                     "nxphlite-v3_default", "omnibus-f4sd_default"],
+                     "intelaerofc-v1_default", "gumstixaerocore2_default", "auavx21_default", "avx-v1_default", "bitcrazecrazyflie_default", "airmindmindpx-v2_default",
+                     "nxphlite-v3_default", "omnibusf4sd_default"],
             image: docker_images.nuttx,
             archive: true
           ]
 
           def nuttx_builds_other = [
-            target: ["px4-same70xplained-v1_default", "px4-stm32f4discovery_default", "px4cannode-v1_default",
-                     "px4esc-v1_default", "px4nucleoF767ZI-v1_default", "s2740vc-v1_default"],
+            target: ["atmelsame70xplained_default", "stm32f4discovery_default", "px4cannode-v1_default",
+                     "px4esc-v1_default", "stmnucleo-F767ZI_default", "thiemars2740vc-v1_default"],
             image: docker_images.nuttx,
             archive: false
           ]
 
           def rpi_builds = [
-            target: ["posix_rpi_cross", "posix_bebop_default"],
+            target: ["emlidnavio2_cross", "parrotbebop_default"],
```

Comment:
The main reason it's been kept like this so far is to minimize migration pain.

     px4fmu-v5_default -> px4_fmu-v5_default

I'd still like to split, but we'll need everyone onboard so that we can do a coordinated documentation update.

Reply:
My lets fix this once and for all. Let's move forward with the name change.  Provide an "is" "was" list in the commit and we can all adjust or fix what it breaks.